# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../arphound/gcc4.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- arphound-1.3.1/src/Makefile.orig	2006-04-28 15:46:56.000000000 +0200
+++ arphound-1.3.1/src/Makefile	2006-04-28 15:46:23.000000000 +0200
@@ -11,7 +11,7 @@
 ob: $(OBJ_SRC:=.o)
 
 .C.o:
-	$(CPP) $(C_FLAGS) $(DEFINES) -c $<
+	$(CXX) $(C_FLAGS) $(DEFINES) -c $<
 
 .c.o:
 	$(CC) $(C_FLAGS) -c $<
--- arphound-1.3.1/shared/bsocket_l.h.orig	2006-04-28 15:38:27.000000000 +0200
+++ arphound-1.3.1/shared/bsocket_l.h	2006-04-28 15:39:18.000000000 +0200
@@ -60,7 +60,7 @@
   unsigned long SlowSend(const char * source,unsigned long len);
   unsigned long Send(const char *Source);
   void GetLocalIp(char *ip); // renvoit l'ip locale d'une sock
-  bool bSocket::Bind(int Port,char * interface=NULL); // a faire avant createrecvloop udp
+  bool Bind(int Port,char * interface=NULL); // a faire avant createrecvloop udp
   // boucle creee dans un nouveau thread, cree une nouvelle
   // bsocket a chaque connection sur le port, et appelle OnConnect
   // attention, 0nConnect doit retourner RAPIDEMENT pour liberer le thread
--- arphound-1.3.1/shared/Makefile.orig	2006-04-28 15:47:52.000000000 +0200
+++ arphound-1.3.1/shared/Makefile	2006-04-28 15:48:02.000000000 +0200
@@ -9,7 +9,7 @@
 ob: $(OBJ_SHARED:=.o)
 
 .C.o:
-	$(CPP) $(C_FLAGS) -c $<
+	$(CXX) $(C_FLAGS) -c $<
 
 .c.o:
 	$(CC) $(C_FLAGS) -c $<
--- arphound-1.3.1/Makefile.common.orig	2006-04-28 15:41:25.000000000 +0200
+++ arphound-1.3.1/Makefile.common	2006-04-28 15:45:59.000000000 +0200
@@ -24,9 +24,9 @@
 
 # Build
 .SUFFIXES = .C .c .o
-CPP = g++
+CXX = g++
 CC = gcc
-C_FLAGS = -O3 -Wno-deprecated -w
+C_FLAGS = -O3 -w
 OBJ_SHARED = lc dnsquery bsocket_l log pidfile parseconfig
 OBJ_SRC = arphound
 
--- arphound-1.3.1/Makefile.orig	2006-04-28 15:47:21.000000000 +0200
+++ arphound-1.3.1/Makefile	2006-04-28 15:47:32.000000000 +0200
@@ -13,7 +13,7 @@
 	@cd shared && make
 
 arphound: binary 
-	$(CPP)  -o arphound $(OBJ_SRC:%=src/%.o) $(OBJ_SHARED:%=shared/%.o) -lpcap -lpthread
+	$(CXX)  -o arphound $(OBJ_SRC:%=src/%.o) $(OBJ_SHARED:%=shared/%.o) -lpcap -lpthread
 
 install:
 	@echo -n "Installing ArpHound... "
